manifests/cf-manifest/cloud-config/010-cf-networks.yml

```diff
@@ -1,7 +1,6 @@
 ---
-
 networks:
-- name: cf1
+- name: cf
   subnets:
     - range: 10.0.16.0/24
       reserved:
```

Comment:
Btw. this has been there before, in this PR we only rewrite BOSH semantics, so I'll fix this in a new commit (or potentially a new PR - as this change takes a while to test).